spec/AutomatedLinkAcceptanceProxy.yaml

```diff
@@ -1017,21 +1023,6 @@ paths:
                           alarm-type-qualifier: ''
                           alarm-severity: 'alarms-1-0:SEVERITY_TYPE_CRITICAL'
           headers:
-            x-correlator:
-              schema:
```

Comment:
Here, we could see header parameters are removed. Could you please kindly let me know the reason behind this. (asking just for understanding purpose)

Reply:
Headers do not need to be sent to Applications outside the SDN Domain since these cannot obtain anything from it